### DISCUSSION TITLE
What is asm_sysvec_apic_timer_interrupt in cache-misses flamegraph?

### DISCUSSION THREAD

**Discussion by hokamc** [async-profiler/async-profiler#985] [Q&A] [answered]
<img width="989" alt="image" src="https://github.com/user-attachments/assets/d840bd7e-f4d3-4317-83bf-7b536000b82a">

Does the green part indicate a cache miss or that the OS scheduler time slice has ended?

**Comment by apangin** [comment, marked as answer]
As the name suggests, `asm_sysvec_apic_timer_interrupt` is the kernel handler for APIC timer interrupts.
The flame graph shows combined stack: Java part (green) + kernel (brown). The topmost frame is a function where cache miss occurs.
